### PR TITLE
Performance: Add next image component for performance tasks

### PR DESCRIPTION
**Pull request by alejandrocabriales**
# Ticket
https://trello.com/c/5eYTX48I/116-issues-pending-of-r1-v-20

# Description

# Actions

delete-image-background-add-gradient

Delete-unnecessary-image-add-gradient-property

### CODE REVIEW

**Review by z3phyro — APPROVED** [review]
LGTM

File: src/stories/components/footer/footer-link.tsx [hunk 1]
```diff
@@ -22,9 +22,9 @@ export const FooterLinks = ({ links, styleLinks }: CuTableColumnLinksProps) => {
     {links.map((link, i) => <Box
       key={`link-${i}`}
       sx={{ mr: `${link.spacingsRight ?? 0}px` }}>
-      <CustomPopover
+      {/* <CustomPopover
```
Comment (z3phyro) [inline src/stories/components/footer/footer-link.tsx]: Is this removed in a commit after? We shouldn't keep the comment there

**Review by z3phyro — CHANGES_REQUESTED** [review]

File: src/stories/components/footer/footer-link.tsx (hunk 1 above)
Comment (mact200590) [reply src/stories/components/footer/footer-link.tsx]: Yes , the commnet is delete in this commit:
fix-comment

**Review by z3phyro — APPROVED** [review]
LGTM

**Pull request merged**